### PR TITLE
Content layout1

### PR DESCRIPTION
**Pull request by marcalmon**
## Summary

- Completed the card of the content layout

## Screenshots

Screenshot from 2019-06-12 16-24-41

Screenshot from 2019-06-12 16-25-10

## Trello Card

https://trello.com/c/sExL41uP/23-content-layout

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -8,6 +8,65 @@
     <link rel="stylesheet" type="text/css" href="../application.css">
 </head>
 <body>
-    <h1 class="title">Front-End Tools</h1>
+	<h1 class="title">Hello World</h1>
+	<div class="content-tech">
+		<div class="tech">
+			<img src="../maquetado/assets/react_icon.svg" alt="" class="tech-image">
```
Comment (pabloccid) [inline maquetado/index.html]: You can simplify the route of the images here. You can try `assets/react_icon.svg` or `./assets/react_icon.svg`
This comment apply to every image

File: maquetado/index.html [hunk 2]
```diff
@@ -8,6 +8,65 @@
     <link rel="stylesheet" type="text/css" href="../application.css">
 </head>
 <body>
-    <h1 class="title">Front-End Tools</h1>
+	<h1 class="title">Hello World</h1>
```
Comment (pabloccid) [inline maquetado/index.html]: Here you should have the title with the image behind that you developed in a previous branch

File: scss/tech.scss [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    flex-direction: column;
+    justify-content: center;
+    margin: 20px;
+    padding: 0px 25px;
+    position: relative;
+
+    &::after {
+      @include card-size;
+      content: ' ';
+      border: 1px solid $green;
```
Comment (gabrielizalo) [inline scss/tech.scss]: Fix attributes sort of `border` and `content`.

File: scss/tech.scss [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      left: -10px;
+      position: absolute;
+      top: -10px;
+      z-index: -1;
+    }
+
+    .tech-image {
+      margin-bottom: $space-between-texts;
+      width: $tech-logo-size;
+      height: $tech-logo-size;
```
Comment (gabrielizalo) [inline scss/tech.scss]: Fix attributes sort of `width` and `height`.

File: scss/titles.scss [hunk 5]
```diff
@@ -2,6 +2,9 @@
 @import 'sizes';
 
 .title {
-	color: $color-title;
+	color: $black;
 	font-size: $size-title;
+	padding-left: 300px;
+	padding-top: 100px;
+	padding-bottom: 10px;
```
Comment (gabrielizalo) [inline scss/titles.scss]: Use the Padding shorthand Property: `padding: 100px ??px 10px 300px;`, changing `??` for the proper value.

File: scss/titles.scss [hunk 6]
```diff
@@ -2,20 +2,20 @@
...
-	.title {
-		color: black;
-		font-size: $text-big;
-		position: absolute;
-		left: 75px;
-	}
+  .title {
+    color: black;
+    font-size: $text-big;
+    position: absolute;
```
Comment (gabrielizalo) [inline scss/titles.scss]: Here `position` is after `left`. Fix for the next PR.

**Pull request merged**